tests/unit/test_tempo.py

```diff
@@ -22,12 +22,17 @@
             {
                 "jaeger": {
                     "protocols": {
-                        "grpc": None,
-                        "thrift_http": None,
+                        "grpc": {"endpoint": "0.0.0.0:14250"},
```

Comment:
to make the test less fragile might we one day decide to change the ports:

```suggestion
                        "grpc": {"endpoint": f"0.0.0.0:{Tempo.all_ports['jaeger_grpc']}"},
```